pywhispercpp/__init__.py

```diff
@@ -4,5 +4,8 @@
 
 lib_dir = os.path.join(os.path.dirname(__file__), 'lib/*')
 
+# Append lib dir to PATH
+os.add_dll_directory(os.path.join(os.path.dirname(__file__), 'lib'))
```

Comment:
Is this needed ? 
Loading the files with ctypes works without issues, on my end ?

Reply:
Without it failed to load the library. that's reasonable it looks for whisper.dll and it should be on PATH.
That's adding the folder to path

Reply:
The `whisper.dll` will be loaded with the `pyd` file basically.
Not sure why it works on Linux and the VM yesterday without it! 
It's OK, I'll merge this for now!

Reply:
Why whisper.dll should loaded with `pyd`? I think that it's basically tries to load such as _Pseudo_: LoadLibrary('whisper.dll`) and that means it needs to be in the PATH

Reply:
I thought this is what `ctypes.CDLL` is for, it loads the dlls to the path! 
Tha piece of code goes through all the files in the lib folder and add them to path. when `pyd` is loaded it will look for the shared library. 
I think you told me it worked on Linux last time on your side as well, have you added anything to LD_LIBRARY_PATH ? it worked on the Windows VM as well!

Reply:
It loads the `pyd` which is sort of DLL file but then that `pyd` file depends on another `dll` file (`whisper.dll`) and it doesn't load it relatively it basically tells the system 'give me the file whisper.dll' and then search in PATH folders..

Reply:
This is failing on Linux now,  "AttributeError: module 'os' has no attribute 'add_dll_directory'" !